tests/linalg/operator_market.py

```diff
@@ -18,7 +18,7 @@
     'psd_scalarmul',
     'selfadj_hessian',
     'selfadj_tridiagonal',
-    'square_big',  # skipped by default
+    # 'square_big',  # skipped by default
```

Comment:
@mfinzi , I'm gonna move this case outside of this unit tests

Reply:
sounds good

Reply:
I will add a decorator tomorrow to manage these tricky test cases